go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/ProtonMail/go-crypto v1.1.0-alpha.0 // indirect
 	github.com/agext/levenshtein v1.2.3 // indirect
 	github.com/apparentlymart/go-textseg/v15 v15.0.0 // indirect
+	github.com/ciscoecosystem/aci-go-client v1.54.0 // indirect
```

Comment:
Why are we adding v1 of go-client as a dependency?

Reply:
Removed the old dependency by using go mod tidy